chapters/tr/chapter1/5.mdx

```diff
@@ -0,0 +1,17 @@
+# Encoder modelleri
+
+<Youtube id="MUqNwgPjJvQ" />
+
+Encoder modelleri Transformer modellerinin sadece encoder kısmını kulanır.Her aşamada, attention katmanları ilk cümlenin bütün kelimelerine erişir. Bu modeller genellikle çift yönlü attention olarak nitelendirilir ve genellikle *auto-encoding models* olarak adlandırılır.
+
+Bu modellerin öneğitimi genellikle verilen cümleyi bozmaya yöneliktir (örnek olarak, içindeki rastgele kelimeleri maskeleyerek) ve model ilk cümleyi bulma veya yeniden oluşturma ile görevlendirilir.
+
+Encoder modelleri cümle sınıflandırma, varlık tanıma (ve daha genel olarak sözcük sınıflandırma) ve çıkarımsal soru yanıtlama gibi cümlenin tam anlaşılmasını gerektiren görevler için uygundur.

```

Comment:
```suggestion
Encoder modelleri cümle sınıflandırma, varlık tanıma (daha spesifik olarak sözcük sınıflandırma) ve extractive soru yanıtlama gibi cümlenin tam anlaşılmasını gerektiren görevler için uygundur.

```
Extractive QA doesn't really have a translation and despite "çıkarımsal" being a good fit I believe that it wouldn't make sense to the reader. so I think leaving it as it is better.

Reply:
Definitely agree, I was a little confused when saying "çıkarımsal", We can just leave as Extractive Q/A, because everybody knows what Q/A means :)